content/en/docs/reference/controllers/node.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Node controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The node controller is a Kubernetes master component which manages various
+aspects of nodes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Controller behavior
+
+The node controller has multiple roles in a node's life. The first is assigning a
+CIDR block to the node when it is registered (if CIDR assignment is turned on).
+
+The second is keeping the node controller's internal list of nodes up to date with
+the cloud provider's list of available machines. When running in a cloud
+environment, whenever a node is unhealthy, the node controller asks the cloud
+provider if the VM for that node is still available. If not, the node
+controller deletes the node from its list of nodes.
+
+The third is monitoring the nodes' health. The node controller is
+responsible for updating the NodeReady condition of NodeStatus to
+ConditionUnknown when a node becomes unreachable (i.e. the node controller stops
+receiving heartbeats for some reason, e.g. due to the node being down), and then later evicting
+all the pods from the node (using graceful termination) if the node continues
+to be unreachable. (The default timeouts are 40s to start reporting
+ConditionUnknown and 5m after that to start evicting pods.) The node controller
+checks the state of each node every `--node-monitor-period` seconds.
+
+The node controller looks at the state of all nodes in the cluster when making a
+decision about pod eviction.
+
+In most cases, node controller limits the eviction rate to
+`--node-eviction-rate` (default 0.1) per second, meaning it won't evict pods
+from more than 1 node per 10 seconds.
```

Comment:
@sftim imo i find that we could maybe reword it a bit like 

_won't evict pods from more than 1 node on a 10 seconds window_